remark/detect-bad-links.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path')
+const visit = require('unist-util-visit')
+
+const shouldLinkGoToDocsCozyIO = url => {
+  const [base, fragment] = url.split('#')
+  return base.startsWith('https://github.com/cozy/') && base.endsWith('.md') && base.includes('/blob/master')
+}
+
+const updateURLToDocsCozyIO = url => {
+  const [base, fragment] = url.split('#')
+  const newURL = base
+    .replace('https://github.com/cozy/', 'https://docs.cozy.io/en/')
+    .replace('/blob/master', '')
+    .replace(/\.md$/, '')
+    .replace('cozy-stack/docs', 'cozy-stack')
+    .replace('cozy-konnector-libs/packages/cozy-konnector-libs/docs', 'cozy-konnector-libs')
```

Comment:
don't you need to handle /README/ ?

Reply:
Yes, I did that by hand afterwards, fixing..